### ISSUE TITLE
Blah blah blah

### ISSUE THREAD

**Issue by dan1410** [dan1410/Projekt_STD#1]
Czyżby to moduł Issues był do komunikacji?

**Comment by Radi175** [comment]
:)

**Comment by dan1410** [comment]
Tajest!

**Comment by tarnowski-git** [comment]
Czyli już wszyscy mamy odhaczone pierwsze polecenie

**Comment by dan1410** [comment]
Pytanie, jak dalej podzielić się robotą

**Comment by dan1410** [comment]
Okej, wysłana baza pod zadania na oba repozytoria

**Comment by Radi175** [comment]
a gdzie sa te biblioteki do dodania???

**Comment by dan1410** [comment]
To chodzi po prostu, żeby potem dodać linię #include "stdafx.h"
Ale cały czas szukam, jak stworzyć ten pull request

**Comment by Radi175** [comment]
https://help.github.com/articles/creating-a-pull-request/

**Comment by dan1410** [comment]
A, chwilka, chyba łapię

**Comment by dan1410** [comment]
Chcecie zrobić drugie...? Bo pierwsze będzie zaraz odhaczone

**Comment by Radi175** [comment]
ok

**Comment by tarnowski-git** [comment]
w sensie, pierwsza część zad 2 jest już zrobiona?

**Comment by Radi175** [comment]
Do gałęzi Projekt_STD nanieś odpakowaną zawartość dp1.zip czy - Stwórz Pull request domagające się aktualizacji okna informacji

**Comment by dan1410** [comment]
zadanie drugie jest już zrobione *całe*

**Comment by tarnowski-git** [comment]
Aaa. OK.

**Comment by dan1410** [comment]
Z tworzeniem pull requesta jest tak:
trzeba utworzyć nową gałąź: (wpisać jakąś nazwę)
![obraz](https://user-images.githubusercontent.com/34545189/33997376-7219b600-e0e4-11e7-8fb2-d7950a4a046b.png)

**Comment by dan1410** [comment]
Potem będąc na niej, trzeba zedytować pliki

**Comment by dan1410** [comment]
A potem, jak się wejdzie w "Pull requests>New pull request", to trzeba wybrać powównanie tej nowej gałęzi z gałęzią master
![obraz](https://user-images.githubusercontent.com/34545189/33997459-bf77e156-e0e4-11e7-8b26-7d894768b4a3.png)

**Comment by Radi175** [comment]
kuwa uczylem sie tego w domu na bashu i na tej stronce nie ogarniam xd

**Comment by dan1410** [comment]
W sumie z poziomu basha przydaje się bardziej niż z poziomu strony :)

To ja jeszcze zajmę się tymi wpisami do projektu

**Comment by Radi175** [comment]
a w tym 2 zadaniu jest zrobiony ten pull request?

**Comment by dan1410** [comment]
Zadanie drugie zrobiłem całe, razem z tymi dwoma pull requestami

**Comment by Radi175** [comment]
aa dobra,widze,to juz w closed jest

**Comment by dan1410** [comment]
"Zaktualizuj powyższy projekt tak, aby w oknie informacje pojawiło się wasze imiona i nazwiska" -> zrobione

**Comment by dan1410** [comment]
Albo i nie, cholera nie ogarniam, o które okno z informacjami chodzi? Jest tam coś w kodzie takiego?

**Comment by tarnowski-git** [comment]
Ok. A czy mamy "Stwórz Pull request domagające się aktualizacji okna informacji" ?

**Comment by Radi175** [comment]
![to](https://user-images.githubusercontent.com/34344333/33997864-17dd2d28-e0e6-11e7-86c8-5f49f6d60505.png)

a z tymi informacjami to nie chodzi o to?

**Comment by tarnowski-git** [comment]
Bo przyznam, że nie wiem jak to sprawdzić z mojego poziomu :/

**Comment by dan1410** [comment]
Kurczę, a może to chodzi o to, dobry pomysł

**Comment by dan1410** [comment]
Do skoro już się do niego dogrzebałem, to ogarnę to okno

**Comment by dan1410** [comment]
(Chyba, że Wy chcecie je ogarnąć)

**Comment by tarnowski-git** [comment]
Jeśli pokierujesz jak, to nie ma problemu

**Comment by dan1410** [comment]
Okej, tylko patrzę, że pliki do zadania 3 są wrzucone podwójnie, więc uwaga na to

**Comment by Radi175** [comment]
zrobilem pull requesta o wersji ale nie wiem czy dokladnie o to chodzilo xd

**Comment by dan1410** [comment]
Dokładnie o to! 👍

**Comment by dan1410** [comment]
To jeszcze ta ostatnia zmiana: 
> trzeba zmienić branch na "test"
> przejść do pliku 'resources.rc'
> zedytować i zapisać
> i potem w zakładce pull requests trzeba stworzyć nowy, i wybrać gałęzie jak na screenie gdzieś tam hen w górze

**Comment by dan1410** [comment]
a nie, to plik 'dp1.rc'

**Comment by Radi175** [comment]
a gdzie ty tam masz plik 'resources.rc xd

**Comment by dan1410** [comment]
No właśnie się ogarnąłem :D

**Comment by dan1410** [comment]
Ale to może @tarnowski-git chcesz to przećwiczyć?

**Comment by Radi175** [comment]
ale co ty chcesz tam edytowac?

**Comment by tarnowski-git** [comment]
Mógłbym przećwiczyć.

**Comment by dan1410** [comment]
Tam jest jedna linijka, zaraz puszczę screena

**Comment by dan1410** [comment]
![obraz](https://user-images.githubusercontent.com/34545189/33998628-667bf2aa-e0e8-11e7-9b1c-735c973ed049.png)

**Comment by dan1410** [comment]
Albo druga z opisu, trzeba którąś z nich zmienić

**Comment by tarnowski-git** [comment]
@dan1410 czyli trzeba wejść w ten plik, dokonać zmiany i tę zmianę jakoś zapisać?

**Comment by dan1410** [comment]
Tak, ale wszystko z poziomu przeglądarki:
![obraz](https://user-images.githubusercontent.com/34545189/33998734-bb4466a0-e0e8-11e7-9622-fe10c537d705.png)
A potem kliknąć "Commit changes"

**Comment by tarnowski-git** [comment]
Dajcie mi chwilkę

**Comment by dan1410** [comment]
Spokojnie :)

**Comment by dan1410** [comment]
To ja jeszcze przelecę w międzyczasie, czy czegoś nie pominęliśmy

**Comment by Radi175** [comment]
masz 2 min albo wyrzucamy z grupy

**Comment by Radi175** [comment]
:D

**Comment by tarnowski-git** [comment]
Coś stworzyłem, ale dodałem jako nowy branch

**Comment by dan1410** [comment]
To dokładnie tak ma być
Bo w tych pull requestach chodzi o to, że zmiany są dokonywane na gałęziach (branchach) testowych, i dopiero potem to się włącza do głównego kodu (`master`)

**Comment by dan1410** [comment]
Dokładnie o to chodziło :)
To już to tylko aprobuję i robota skończona :)

**Comment by tarnowski-git** [comment]
Czyli master aprobuje zmiany jakie dokonują pozostali? To jest cała idea GitHuba? :)

**Comment by dan1410** [comment]
**Tadam** 🎆 🎊 (i jeszcze mnóstwo innych dziadowskich ikonek)

**Comment by Radi175** [comment]
to juz wszystko mamy?

**Comment by dan1410** [comment]
Nie, master to tak jakby główny "folder" z kodem, zaraz jak już będzie można gadać to Ci to lepiej opiszę :)
